docs/Error Types Class Diagram.puml

```diff
@@ -0,0 +1,104 @@
+@startuml
+
+title PSC Filing API: Error Class Structure
+skinparam BackgroundColor GhostWhite
```

Comment:
Text looks faded to me, so I'd suggest adding this:
`skinparam defaultFontColor black`

Reply:
Updated as requested. Also made background colour pure white.